docs/en_US/CommunitySharings/NNI_colab_support.md

```diff
@@ -0,0 +1,44 @@
+
+# Use NNI on Google Colab
+NNI can easily run on Google's colab platform. However, colab doesn't expose its public IP and ports, so by default you can not access NNI's web ui on colab. To solve this, you need a reverse proxy software like `ngrok` or `frp`. This tutorial will show you how to use ngrok to get access to NNI's web ui on colab.
+
+## How to Open NNI's WebUI on Google's Colab
+
+1. Install required packages and softwares.
+
+
+```
+! pip install nni # install nni
+! wget https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip # download ngrok and unzip it
+! unzip ngrok-stable-linux-amd64.zip
+! mkdir -p nni_repo
+! git clone https://github.com/microsoft/nni.git nni_repo/nni # clone NNI's offical repo to get examples
+```
+
+2. Register a ngrok account [here](https://ngrok.com/), then connect to your account using your authtoken.
+
+
+```
+! ./ngrok authtoken <your-authtoken>
+```
+
+3. Start an NNI trail example on a port greater than 1024, then start ngrok with the same port.
+
+
+```
+! nnictl create --config nni_repo/nni/examples/trials/mnist-pytorch/config.yml --port 5000 & # if you want to use gpu, make sure gpuNum >= 1 in config.yml
+get_ipython().system_raw('./ngrok http 5000 &')

```

Comment:
why not use `./ngrok http 5000 &` directly?

Reply:
If use `! ./ngrok http 5000 &` directly, the cell running this command will stuck. Not sure why this happened. Use get_ipython() will solve this problem.

Reply:
fixed.